bridge/discord/handlers.go

```diff
@@ -103,7 +103,16 @@ func (b *Bdiscord) messageCreate(s *discordgo.Session, m *discordgo.MessageCreat
 	// if we have embedded content add it to text
 	if b.GetBool("ShowEmbeds") && m.Message.Embeds != nil {
 		for _, embed := range m.Message.Embeds {
-			rmsg.Text = rmsg.Text + "embed: " + embed.Title + " - " + embed.Description + " - " + embed.URL + "\n"
+			if embed.Title != "" {
+				rmsg.Text += "embed: " + embed.Title
```

Comment:
Is it possible for `embed.Title` to be empty but for the other fields to be set? If so, should `embed: ` be missing whilst the others are appended?

Reply:
Yeah probably :-)
I hate these kind of checks, do you know a more elegant way to handle this?

Reply:
I hate them too. You could probably range over a string array, keeping track of whether you've appended something yet, and append the prefix / str / separator when needed.

But it's probably a bit unnecessary for appending three strings.

In the future with Go contracts we could probably do something like

```go
"embed: " + strings.Join(slices.RemoveZero([]string{embed.Title, embed.Description, embed.URL}), " - ")`
```

Reply:
Went all the way now ;-)